zhmccli/_cmd_cpc.py

```diff
@@ -349,28 +349,31 @@ def dpm_export(cmd_ctx, cpc, **options):
 
 @cpc_group.command('dpm-import', options_metavar=COMMAND_OPTIONS_METAVAR)
 @click.argument('CPC', type=str, metavar='CPC', required=True)
-@click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
-              expose_value=False,
-              help='Skip prompt to confirm import of the DPM configuration.',
-              prompt='Are you sure you want to import the DPM configuration '
-              'into the CPC, replacing its current DPM configuration ?')
+@click.option('-y', '--yes', is_flag=True,
+              help='Skip prompt to confirm import of the DPM configuration.')
 @click.option('--dpm-file', '-d', metavar='FILE', type=str, required=True,
               help='Path name of the DPM configuration file to be used.')
 @click.option('--dpm-format', '-f', type=click.Choice(DPM_FORMATS),
               required=False, default=DEFAULT_DPM_FORMAT,
               help='Controls in which data format the DPM configuration file '
               'is read. Default: {d}.'.format(d=DEFAULT_DPM_FORMAT))
-@click.option('--preserve-uris', is_flag=True, required=False,
-              help='Controls whether the URIs and IDs of objects in the DPM '
-              'configuration file are preserved or ignored and newly '
-              'generated. Default: Ignored and newly generated.')
-@click.option('--preserve-wwpns', is_flag=True, required=False,
-              help='Controls whether the WWPNs of HBAs in the DPM '
-              'configuration file are preserved or ignored and newly '
-              'generated. Default: Ignored and newly generated.')
+@click.option('--preserve-uris/--generate-uris',
```

Comment:
Don't you need to specify separate click options?

Reply:
This is on purpose, so they end up "together" and one can check preserve-uris for "both" cases.